### DISCUSSION TITLE
Create functions problem

### DISCUSSION THREAD

**Discussion by igorbrai** [osm-search/Nominatim#2353] [Q&A]
Nominatim=3.7.0
Psql 9.5
Hi i am trying to install the functions because during index it failed, stating some function is missing, so i re-run `nominatim refresh --functions --enable-debug-statements` and i get this error: 
 

`psycopg2.errors.SyntaxError: syntax error at or near "%"
LINE 3225:       RAISE WARNING 'Updating names for country '%' with: %'...`

any ideas ?

**Comment by lonvia** [comment]
Just don't use `--enable-debug-statements`. It's really just for developers and broken half the time.

**Reply by igorbrai** [reply]
Thanks for the reply, even without this ,everything seems to work but then when re-indexing it fails again due to missing function
